### ISSUE TITLE
Target size of "Copy collaboration link", "Stop collaboration session" controls is less than 24x24px: A11y_Visual Studio Code_"Session Details ", "Help" sections_Target Size

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#42607]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

--- Environment Details:

Application Name: Visual Studio Code
Edition Windows 11 Enterprise
Version 22H2
OS build 22621.3372
Visual Studio code
Version 1.87.2

--- Github Tags:

-A11yTCS; -A11ySev2; -WCAG2.5.8; -DesktopApp; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Jan2024; -FTP; -Benchmark; -Win32; -A11yMAS;

--- Pre-requisites:

Go to mouse settings and TAB to Additional mouse settings.
TAB to Browse and press ENTER.
[24x24.cur](https://microsoft-my.sharepoint.com/:u:/p/scottohara/ETmopeGhePhEqauAHj65FEsBotnZ1cozkonud-WfpYohog?e=B40D09&isSPOFile=1) download it and Apply.

--- Repro Steps:

Open visual studio code application.
Install "Live share" extension.
TAB till "Share read/write" control and hit ENTER.
TAB till "Session Details ", "Help" sections.
Observe that Target size of "Copy collaboration link", "Stop collaboration session" controls is less than 24x24px.

--- Actual Result:

Target size of "Copy collaboration link", "Stop collaboration session" controls is less than 24x24px.

--- Expected Result:

Target size of "Copy collaboration link", "Stop collaboration session" controls should be at least 24x24 px to ensure that they are accessible to all users, including those with visual impairments.

--- User Impact:

People with visual impairments may have difficulty using the "Copy collaboration link", "Stop collaboration session" controls if they are too small. The target size of 24x24 px ensures that the buttons are large enough to be seen and used easily. Without this size, people with visual impairments may not be able to access the content.

![Target size](https://github-com/microsoft/vscode/assets/115216656/0620be09-e4c2-4b16-9ac8-9af834c68520)
